### ISSUE TITLE
April meetup

### ISSUE THREAD

**Issue by aaronj1335** [austinjavascript/austinjavascript.github.io#42]
Tuesday April 18<sup>th</sup>.

- [ ] Speaker: @markedwardsims
- [ ] Sponsor:
- [ ] Venue: will confirm at the beginning of April

**Comment by markedwardsims** [comment]
Hey guys, can you tell me a little bit about sponsorship? Maybe I can get p202 to sponsor the April meetup.

**Comment by aaronj1335** [comment]
For sure, I'll follow up over email, thanks!

**Comment by markedwardsims** [comment]
Hey guys! We're getting closer, so I just wanted to make sure everything is square for Tuesday April 18th. If you're good with it, my company would like to promote it through social media. Looking forward to it!

**Comment by aaronj1335** [comment]
Yep, we're still locked in to have you speak, thanks for confirming!

**Comment by markedwardsims** [comment]
Do we have a venue booked? I've got my travel arrangements made, but need to know where to go. Also, what time would you like me there? Thanks!
